README.md

```diff
@@ -81,3 +81,11 @@ When making changes to measures-data, include tests in the tests directory and m
 ```
 npm run test
 ```
+
+### Additional measures
+
+`util/additional-measures.json` includes data objects which are necessary for scoring but are not MIPS measures. At this time `util/additional-meausures.json` includes:
```

Comment:
Thanks for catching that ! https://github.com/CMSgov/qpp-measures-data/pull/18/commits/695b639e6639eb87ddc066e9cfc723b8b715215d